scripts/tablegen/values.yaml

```diff
@@ -0,0 +1,278 @@
+# Description: Gateways Configuration, refer to the charts/gateways/values.yaml
```

Comment:
shouldn't need to check this file in.  As you can tell now from looking at the script it can use some improvement :)